hack/e2e.go

```diff
@@ -145,7 +145,7 @@ func (t tester) getKubetest(get bool, old time.Duration) (string, error) {
 	}
 	log.Print("Updating kubetest binary...")
 	if err = t.wait("go", "get", "-u", "k8s.io/test-infra/kubetest"); err != nil {
-		return "", err // Could not upgrade
+		return "", fmt.Errorf("go get -u k8s.io/test-infra/kubetest: %v"), err // Could not upgrade
```

Comment:
could put command in a []string to avoid a future change making the error message misleading, but it's probably fine